{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"missingInstance"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/mgenetwork\/networkerror\/missinginstance"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/MGENetwork\/documentation\/MGENetwork\/NetworkError\/missingInstance","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"The instance does not exist anymore."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"missingInstance"}],"title":"NetworkError.missingInstance","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:10MGENetwork12NetworkErrorO15missingInstanceyA2CmF","modules":[{"name":"MGENetwork"}]},"hierarchy":{"paths":[["doc:\/\/MGENetwork\/documentation\/MGENetwork","doc:\/\/MGENetwork\/documentation\/MGENetwork\/NetworkError"]]},"references":{"doc://MGENetwork/documentation/MGENetwork":{"role":"collection","title":"MGENetwork","abstract":[],"identifier":"doc:\/\/MGENetwork\/documentation\/MGENetwork","kind":"symbol","type":"topic","url":"\/documentation\/mgenetwork"},"doc://MGENetwork/documentation/MGENetwork/NetworkError/missingInstance":{"role":"symbol","title":"NetworkError.missingInstance","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"missingInstance"}],"abstract":[{"type":"text","text":"The instance does not exist anymore."}],"identifier":"doc:\/\/MGENetwork\/documentation\/MGENetwork\/NetworkError\/missingInstance","kind":"symbol","type":"topic","url":"\/documentation\/mgenetwork\/networkerror\/missinginstance"},"doc://MGENetwork/documentation/MGENetwork/NetworkError":{"role":"symbol","title":"NetworkError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"NetworkError"}],"abstract":[{"type":"text","text":"Error occurring upon network tasks."}],"identifier":"doc:\/\/MGENetwork\/documentation\/MGENetwork\/NetworkError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"NetworkError"}],"url":"\/documentation\/mgenetwork\/networkerror"}}}